pom.xml

```diff
@@ -153,24 +155,9 @@
             <artifactId>powsybl-config-test</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-test</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-test-autoconfigure</artifactId>
+            <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
Sure about that ?

Reply:
yes  
>view-source:https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-test/3.3.3/spring-boot-starter-test-3.3.3.pom

Reply:
Then what ? Why should we change ?

Reply:
the pom was using "spring-test" but we are using sprint-boot, so I pass it to "spring-boot-starter-test" like on other servers ; then "spring-boot-test-autoconfigure" is already include by "spring-boot-starter-test" so I remove it from the POM.

Reply:
ok !